.github/workflows/build_wheels_linux_macos.yml

```diff
@@ -0,0 +1,106 @@
+name: Build Linux/MacOS wheels
+
+on:
+  push:
+    branches:
+      - main # just build the sdist & wheel, skip release
+    tags:
+      - "*"
+  pull_request: # also build on PRs touching this file
+    paths:
+      - ".github/workflows/build_wheel.yml"
+      - "ci/*"
+      - "MANIFEST.in"
+      - "pyproject.toml"
+      - "setup.py"
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }} ${{ matrix.platform }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-20.04", "macos-latest"]
+        platform: ["x86_64"]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # TODO:
+      # - name: Cache vcpkg
+      #   uses: actions/cache@v1
+      #   id: vcpkgcache
+      #   with:
+      #     # path: TBD
+      #     key: ${{ runner.os }}-vcpkg
+
+      - name: Install GDAL
+        run: |
+          vcpkg install gdal[core]
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.3.1
+        env:
+          # should be automatically detected?
+          GDAL_INCLUDE_PATH: "/usr/local/include"
```

Comment:
I don't think those will be automatically detected, because vcpkg does not a "system-wide" install AFAIK (so I would find it strange that this gets installed in /usr/local/include, but no idea for linux; on windows this is installed in a vcpkg specific directory)

Reply:
According to https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#environment-variables, it is actually in /usr/local where vcpkg itself gets installed, but not fully sure where exactly it would install binaries.